services/catalog/requirements/_test.txt

```diff
@@ -7,67 +11,67 @@ aiosignal==1.3.1
     # via
     #   -c requirements/_base.txt
     #   aiohttp
-alembic==1.13.1
+alembic==1.14.0
     # via
     #   -c requirements/_base.txt
     #   -r requirements/_test.in
-anyio==4.3.0
+anyio==4.7.0
     # via
     #   -c requirements/_base.txt
     #   httpx
 asgi-lifespan==2.1.0
     # via -r requirements/_test.in
-attrs==23.2.0
+attrs==24.2.0
     # via
     #   -c requirements/_base.txt
     #   aiohttp
     #   jsonschema
     #   pytest-docker
     #   referencing
-certifi==2024.2.2
+certifi==2024.8.30
     # via
     #   -c requirements/../../../requirements/constraints.txt
     #   -c requirements/_base.txt
     #   httpcore
     #   httpx
     #   requests
-charset-normalizer==3.3.2
+charset-normalizer==3.4.0
     # via
     #   -c requirements/_base.txt
     #   requests
 click==8.1.7
     # via
     #   -c requirements/_base.txt
     #   -r requirements/_test.in
-coverage==7.6.1
+coverage==7.6.8
     # via pytest-cov
 docker==7.1.0
     # via -r requirements/_test.in
-faker==29.0.0
+faker==33.1.0
     # via -r requirements/_test.in
-frozenlist==1.4.1
+frozenlist==1.5.0
     # via
     #   -c requirements/_base.txt
     #   aiohttp
     #   aiosignal
-greenlet==3.0.3
+greenlet==3.1.1
     # via
     #   -c requirements/_base.txt
     #   sqlalchemy
 h11==0.14.0
     # via
     #   -c requirements/_base.txt
     #   httpcore
-httpcore==1.0.4
+httpcore==1.0.7
     # via
     #   -c requirements/_base.txt
     #   httpx
-httpx==0.27.0
+httpx==0.28.0
```

Comment:
I suspect this will not work for you because of https://github.com/lundberg/respx/pull/278. At least I had to avoid this upgrade of httpx in the api-server

Reply:
https://github.com/ITISFoundation/osparc-simcore/pull/6846/files#diff-86d5fe588ff2fc7dccb1f4cdd8019d4473146536e88d7a9ede946ea962a91acb